src/oauth-tooling.ts

```diff
@@ -250,6 +249,28 @@ function getAccessToken(options: OAuthConfig, logger?: Logger): Promise<Token> {
   });
 }
 
+function getCredentials(options: OAuthConfig) {
+  let credentialsPromises;
```

Comment:
`const` and in line 256&263 `push(...)`like 267

So we are consitent in that block, in how add stuff.
Or we use the spread operator and create every time a new Array.